galsim/wfirst/wfirst_psfs.py

```diff
@@ -448,3 +437,37 @@ def _find_limits(bandpasses, bandpass_dict):
         if bp.blue_limit < min_wave: min_wave = bp.blue_limit
         if bp.red_limit > max_wave: max_wave = bp.red_limit
     return min_wave, max_wave
+
+def _parse_new_zernikes(infile, out_dir):
```

Comment:
Is there a danger that this function name will become dated?  Maybe just `_parse_zernikes()` or `_parse_zernikes_0715()` would work better?